packages/cli/src/utils/build.js

```diff
@@ -47,6 +45,7 @@ const {
 const checkMissingAppInfo = require('./check-missing-app-info');
 
 const { runCommand, isWindows, findCorePackageDir } = require('./misc');
+const { respectGitIgnore } = require('./ignore');
```

Comment:
```suggestion
const { respectGitIgnore, isBlocklisted } = require('./ignore');
```

See below comment about duplication